scripts/run_check_release_notes

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+git diff origin/master -- doc/news/_preparation_next_release.md | egrep "^\+.+\*\(.+\)\*"
+if [ $? != "0" ]
+then
+	L=git diff origin/master -- doc/news/_preparation_next_release.md | egrep "^\+" | wc -l
```

Comment:
I think you forgot to add [double back ticks or `$()`](https://build.libelektra.org/jenkins/blue/organizations/jenkins/libelektra/detail/PR-2013/5/pipeline/) here:

```
diff: origin/master: No such file or directory
0
scripts/run_check_release_notes: 7: [: Illegal number: 
The release notes were not extended correctly (with  lines).
```

.